dracut/modules.d/99kiwi-lib/kiwi-partitions-lib.sh

```diff
@@ -278,7 +285,9 @@ function get_free_disk_bytes {
 }
 
 function get_partition_table_type {
-    blkid -s PTTYPE -o value "$1"
+    local disk_device="$1"
+    
+    parted --script --machine "${disk_device}" print | grep "^${disk_device}" | cut -d":" -f6
```

Comment:
This change affects all archs and types. We could consider using parted only for `s390`, but I dislike the idea of start adding arch cuts here and there if there is a generic approach.